src/modules/sih/sih_params.c

```diff
@@ -0,0 +1,345 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2013-2019 PX4 Development Team. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ * 3. Neither the name PX4 nor the names of its contributors may be
+ *    used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+
+/**
+ * @file sih_params.c
+ * Parameters for quadcopter X simulator in hardware.
+ *
+ * @author Romain Chiappinelli <romain.chiap@gmail.com>
+ * February 2019
+ */
+
+/**
+ * Vehicle mass
+ *
+ * This value can be measured by weighting the quad on a scale.
+ *
+ * @unit kg
+ * @min 0.0
+ * @decimal 2
+ * @increment 0.1
+ * @group SIH
```

Comment:
Perhaps the group name should be **Simulation In Hardware**. At this point the acronym isn't as well known as SITL or HITL so while eventually SIH might be obvious, right now it is not.

Reply:
I put two underscores to:
- SIH__LAT0
- SIH__LON0
- SIH__H0
- SIH__MU_X
- SIH__MU_Y
- SIH__MU_Z

I would like them to be grouped in the documentation. Is there a better way to do it?

Reply:
I'm thinking of removing those six parameters as they are very specific and don't bring much if changed by the user.